cpu/saml21/include/periph_cpu.h

```diff
@@ -30,10 +30,29 @@ extern "C" {
  * @brief   Mapping of pins to EXTI lines, -1 means not EXTI possible
  */
 static const int8_t exti_config[2][32] = {
+#if defined(CPU_MODEL_SAML21G18A)
```

Comment:
None in the upstream codebase, but having this will avoid someone using RIOT for his own board to have to change this file, or copy it outside of the main source tree.
I can remove if you insist.

Reply:
You're right but what about SAML21E18 ? and SAML21x16 / SAML21x17 ?
I won't oppose to this but I am not convinced. 
If we keep this, you also must add the B variant.